### PR TITLE
Require queue on user create

### PR DESCRIPTION
**Pull request by evelyn9191**

### CODE REVIEW

File: elisctl/user.py [hunk 1]
```diff
@@ -17,7 +17,11 @@ def cli() -> None:
 @cli.command(name="create", short_help="Create user.")
 @click.argument("username")
 @option.password
-@option.queue(related_object="user")
+@option.queue(
+    related_object="user",
+    required=True,
+    help="Queue IDs, that the user will be associated with. Required field.",
```
Comment (KPilnacek) [inline elisctl/user.py]: No need to rephrase the help. This way the help looks like: `Queue IDs, that the user will be associated with. Required field.  [required]`.

btw just a nitpick: the part of english sentence starting with _that_ is not being started with comma (_which_ can be used in such a case). https://ell.stackexchange.com/questions/9154/use-of-commas-before-that

**Review by KPilnacek — CHANGES_REQUESTED** [review]
LGTM, except a tiny nitpick.

File: elisctl/user.py (hunk 1 above)
Comment (evelyn9191) [reply elisctl/user.py]: Thanks for the advice!

**Review by KPilnacek — APPROVED** [review]

**Comment by evelyn9191** [comment]
Closes https://github.com/rossumai/elisctl/issues/81

**Pull request merged**